projects/RabbitMQ.Client/client/impl/ISession.cs

```diff
@@ -73,8 +73,8 @@ internal interface ISession
         void Close(ShutdownEventArgs reason, bool notify);
         bool HandleFrame(in InboundFrame frame);
         void Notify();
-        void Transmit(in OutgoingCommand cmd);
-        void Transmit(IList<OutgoingCommand> cmds);
-
+        void Transmit<T>(in T cmd) where T : struct, IOutgoingCommand;
+        void Transmit<T>(in T cmd, ContentHeaderBase header, ReadOnlyMemory<byte> body) where T : struct, IOutgoingCommand;
```

Comment:
change to `void Transmit<T>(in T cmd, ContentHeaderBase header = default, ReadOnlyMemory<byte> body = default) where T : struct, IOutgoingCommand;` to cover both Transmit cases?